opal/tests/test_core_commandline.py

```diff
@@ -158,7 +158,8 @@ def test_test(self):
         with patch.object(commandline.test_runner, 'run_tests') as rt:
             commandline.test(mock_args)
             mock_args_call = rt.call_args[0][0]
-            self.assertTrue(mock_args_call.userland_here.endswith("opal"))
+            self.assertTrue(mock_args_call.userland_here.endswith("opal"),
+                            "Not running tests from root directory")
```

Comment:
Nice.
I have never read the call signature of `assertTrue` closely enough to notice the second message.

Reply:
All the assertFoo calls take messages. Getting them phrased gramatically is a bit of PITA though as they "look" like comments in the code but aren't displayed in such a context when being run. :)